README.md

```diff
@@ -86,6 +88,37 @@ The merge_core_pattern function is designed to take a patterns DF and cross exam
 The merge_socialDist_by_dates function is designed to merge the social distancing data from a given start_date to a given end_date. The resulting pandas DF will be a DF of all social distancing data from the start_date to the end_date. All Pandas arguments and keywords arguments can be passed into this function.
 * start_date and end_date are strings formated as: "year-month-day"
 
+## cbg_functions
+
+## Usage
+
+```python
+from safegraph_py_functions import cbg_functions as sgpy
```

Comment:
Can you propose a different prefix than `sgpy`, since we already use that for `safegraph_py_functions` above?